packages/discord.js/src/structures/MessageMentions.js

```diff
@@ -19,7 +19,7 @@ class MessageMentions {
    * @type {RegExp}
    * @memberof MessageMentions
    */
-  static UsersPattern = /<@!?(\d{17,19})>/g;
+  static UsersPattern = /<@(\d{17,19})>/g;
```

Comment:
`<@!id>` is also a valid user mention. The issue is it won't show nickname/username but instead both `<@id>` and `<@!id>` will show the same result.

Reply:
They made it seem like they were gonna remove it though https://github.com/discord/discord-api-docs/issues/4734#issuecomment-1086156692

Reply:
No, they're just not going to send it from the client. Making an extremely breaking change like that with such short notice is not really what Discord would do.